Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -156,6 +170,50 @@ YUI.add('ez-platformuiapp', function (Y) {
         },
 
         /**
+         * Add or deletes handleSideViews and handleMainViews from the routes callbacks
+         *
+         * @method _alterRoutesCallback
+         * @protected
+         */
+        _alterRoutesCallback: function() {
+            var app = this;
+
+            Y.Object.each(app.get('routes'), function(route) {
```

Comment:
`routes` value is an array, no ? if so you can iterate with `this.get('routes').forEach()`